documentation-site/pages/guides/theming.mdx

```diff
@@ -157,20 +157,19 @@ As you might expect, `createTheme` is a factory function for Base Web compliant
 ```js
 import {createTheme} from 'baseui';
 
-const theme = createTheme(/* primitives */, /* overrides */);
+const theme = createTheme(/* primitives */, /* overrides */, /* options */);
 ```
 
 #### `primitives`
 
 Our theme object has a lot of properties. Most of these properties are assigned to a subset of reoccurring values. We call these reoccurring values, theme building `primitives`. Primitives are made up mostly of colors, (`primary`, `accent`, etc), gradients of those colors (`primary100`, `primary200`, etc) as well as a `primaryFontFamily`. To see the exact shape for `primitives`, reference the [Flow](https://github.com/uber/baseweb/blob/master/src/themes/types.js#L534) or [Typescript](https://github.com/uber/baseweb/blob/master/src/theme.ts#L668) definitions.
 
-These are passed as the first (and only required) argument for `createTheme`.
+These are passed as the first argument to `createTheme`.
```

Comment:
I think the `required` part was just a misleading wording since it could be just an empty object.

Reply:
It would error if `primitives` and specifically `primitives.primaryFontFamily` were not provided. Updated types and got rid of the potential error so its all optional now.